### ISSUE TITLE
sha256 is slow

### ISSUE THREAD

**Issue by ThomasWaldmann** [MarcoPon/SeqBox#5]
jfyi: sha256 < sha512(-256) < blake2b for 64bit cpus.

**Comment by MarcoPon** [comment]
Yes of course. I initially wanted to use blake2b/s, then I saw that it was supported only by the latest version of Python, so I opted for some more generally available. Anyway, just the fact that it's written in Python means that performance wasn't really the main point of the tool - the whole code is surely not speed optimized, I tried to lean on the clarity/simplicity side if/where possible.  Other hash types can easily be added, and probably will, btw.
